packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -46,6 +52,37 @@ export default async args => {
 
     const result = [];
 
+    console.log(absoluteOutPackageDir);
+
+    function custom(asset, _dir, { assetsPath }) {
+        const { url } = asset;
+        if (url.startsWith("data:")) {
+            const [, mimeType, data] = url.match(/data:([^;]*).*;base64,(.*)/);
+            let extension = mime.extension(mimeType);
+            // Only add extension if we mimeType has associated extension
+            extension = extension ? `.${extension}` : "";
+            const fileHash = crypto.createHash("md5").update(data).digest("hex");
+            const filename = `${fileHash}${extension}`;
+            const filePath = join(absoluteOutPackageDir, filename);
+
+            mkdir("-p", absoluteOutPackageDir);
+
+            writeFile(filePath, data, "base64", err => {
+                if (err) {
+                    if (err.code === "EEXIST") {
```

Comment:
What is this error code for? Maybe it makes sense to add a console log/warn/error here?